test/bigtest/tests/resource-edit-custom-title-test.js

```diff
@@ -188,58 +185,64 @@ describeApplication('ResourceEditCustomTitle', () => {
       });
     });
 
-    describe('entering valid data', () => {
-      beforeEach(() => {
-        return ResourceEditPage
-          .clickAddRowButton()
-          .toggleIsVisible()
-          .dateRangeRowList(0).fillDates('12/16/2018', '12/18/2018')
-          .inputCoverageStatement('Only 90s kids would understand.')
-          .clickAddCustomEmbargoButton()
-          .inputEmbargoValue('27')
-          .inputCustomUrlValue('https://bigtestjs.io')
-          .blurEmbargoValue()
-          .selectEmbargoUnit('Weeks')
-          .blurEmbargoUnit();
+    describe('valid data', () => {
+      beforeEach(async function () {
+        await ResourceEditPage.whenLoaded();
```

Comment:
Weird that this needs `whenLoaded` since `clickAddRowButton` _should_ wait for the button to appear first 🤔